### PR TITLE
parser: Update string representation of stack, making it more readable

### PR DESCRIPTION
**Pull request by shivansh**

### CODE REVIEW

File: internal/parser/gen/golang/parser.go [hunk 1]
```diff
@@ -130,11 +130,16 @@ func (s *stack) String() string {
...
 	for i, st := range s.state {
-		fmt.Fprintf(w, "\t%d:%d , ", i, st)
+		fmt.Fprintf(w, "\t%d: %d , ", i, st)
 		if s.attrib[i] == nil {
 			fmt.Fprintf(w, "nil")
 		} else {
-			fmt.Fprintf(w, "%v", s.attrib[i])
+			switch attr := s.attrib[i].(type) {
+			case *token.Token:
+				fmt.Fprintf(w, "%s", string(attr.Lit))
```
[the remark this answers is not in the document]
Comment (shivansh) [reply internal/parser/gen/golang/parser.go]: Ah, I see ; made the relevant changes.

**Comment by mewmew** [comment]
> Thank you so much for all these cleanups. We really appreciate it. It is great having another contributor on board :)

@awalterschulze, @goccmack I think we should definitely consider inviting @shivansh as an official contributor to the repo. Perhaps this is already done? I can't seem to find where they are listed on GitHub.

Cheers,
Robin

**Comment by awalterschulze** [comment]
I'll plus 1 that :)

**Comment by goccmack** [comment]
@shivansh Would you like to be included as an official contributor to the repo?

**Comment by shivansh** [comment]
> @shivansh Would you like to be included as an official contributor to the repo?

@goccmack It would be an honour!

**Comment by goccmack** [comment]
@shivansh @mewmew @awalterschulze 
Welcome as collaborator shivansh and thank you for your contribution. Please cooperate with memmew and awaltershulze to review commits.

**Comment by shivansh** [comment]
Thanks @goccmack @awalterschulze @mewmew

**Comment by awalterschulze** [comment]
So I think the procedure remains.  That two of us needs to agree for something to be merged.  Even if it is one of us making the pull request, just for efficiency sake.

So in this case I have approved the merge and Shivansh has made the pull request.
So @shivansh can merge this, when he thinks it is ready.
He does not have to wait for another approval.
Or at least that is my proposal, what do the other contributors think?

**Comment by goccmack** [comment]
I agree with @awalterschulze

**Comment by shivansh** [comment]
I see that everyone has agreed (@mewmew gave a thumbs up), so I'm going ahead with the merge.
Thanks!

**Comment by awalterschulze** [comment]
Perfect :)

**Pull request merged**